Update defender-for-storage-introduction.md

Eitan-Shteinberg opened the pull request:
@bmansheim @ElazarK 
Two updates:
1. Added an important note on the new pricing plan
2. Edited the answer in one of the FAQ questions

Eitan-Shteinberg committed:
Update defender-for-storage-introduction.md

Two updates:
1. Added an important note on the new pricing plan
2. Edited the answer in one of the FAQ questions

bmansheim commented on articles/defender-for-cloud/defender-for-storage-introduction.md:
```diff
@@ -23,6 +23,13 @@ Defender for Storage doesn't access the Storage account data and has no impact o
 You can learn more by watching this video from the Defender for Cloud in the Field video series:
 - [Defender for Storage in the field](episode-thirteen.md)
 
+> [!NOTE]
```
Update this to match the note in the storage docs.

bmansheim commented on articles/defender-for-cloud/defender-for-storage-introduction.md:
```diff
@@ -114,7 +121,7 @@ To optimize costs, you might want to exclude specific Storage accounts associate
 ### Can I exclude a specific Azure Storage account from a protected subscription? 
 
-To exclude a specific Storage account when Defender for Storage is enabled on a subscription, follow the instructions in [Exclude a storage account from Microsoft Defender for Storage protections](defender-for-storage-exclude.md). 
+Excluding specific storage accounts from protection is only possible on the per-transaction pricing plan. The per-storage account pricing plan will support exclusion in the future. To exclude a storage account, follow instructions in [Exclude a storage account from a protected subscription in the per-transaction plan](defender-for-storage-exclude.md). 
```
```suggestion
Excluding specific storage accounts from protection is only possible on the per-transaction pricing plan. To exclude a storage account, follow instructions in [Exclude a storage account from a protected subscription in the per-transaction plan](defender-for-storage-exclude.md). 
```

bmansheim reviewed (changes requested):
Looks good. We just want to avoid predicting the future because it can get us into trouble if we don't deliver on it.

Eitan-Shteinberg committed:
Update articles/defender-for-cloud/defender-for-storage-introduction.md

It's important for us to mention that we will support this in the future, is there a way to incorporate it?

Co-authored-by: Ben Mansheim <19514688+bmansheim@users.noreply.github.com>

bmansheim commented on articles/defender-for-cloud/defender-for-storage-introduction.md:
```diff
@@ -114,7 +121,7 @@ To optimize costs, you might want to exclude specific Storage accounts associate
 ### Can I exclude a specific Azure Storage account from a protected subscription? 
 
-To exclude a specific Storage account when Defender for Storage is enabled on a subscription, follow the instructions in [Exclude a storage account from Microsoft Defender for Storage protections](defender-for-storage-exclude.md). 
+Excluding specific storage accounts from protection is only possible on the per-transaction pricing plan. To exclude a storage account, follow instructions in [Exclude a storage account from a protected subscription in the per-transaction plan](defender-for-storage-exclude.md). 
```
```suggestion
Excluding specific storage accounts from protection is only possible on the per-transaction pricing plan. The per-storage account pricing plan will support exclusion in the future. To exclude a storage account, follow instructions in [Exclude a storage account from a protected subscription in the per-transaction plan](defender-for-storage-exclude.md).
```

bmansheim commented on articles/defender-for-cloud/defender-for-storage-introduction.md:
```diff
@@ -23,6 +23,13 @@ Defender for Storage doesn't access the Storage account data and has no impact o
+>
+> Customers using the legacy per-transaction pricing plan will need to proactively migrate to the new per-storage account plan to access these new features and pricing. The legacy per-transaction pricing plan charges based on the number of analyzed transactions in the storage account.
+>
+> For further details, please refer to the [Microsoft Defender for Storage FAQ](https://learn.microsoft.com/azure/storage/common/azure-defender-storage-configure?toc=%2Fazure%2Fdefender-for-cloud%2Ftoc.json&tabs=per-storage-account#faq---microsoft-defender-for-storage-pricing).
```
```suggestion
> Microsoft Defender for Storage customers can now choose to move to a new predictable pricing plan. The pricing model is per-storage account, where high-volume transactions may incur additional overage charges. This new pricing plan will also include all new security features and detections.
>
> Customers using the legacy per-transaction pricing plan need to migrate to the new per-storage account plan to access these new features and pricing. The legacy per-transaction pricing plan charges are based on the number of analyzed transactions in the storage account.
>
> For further details, please refer to the [Microsoft Defender for Storage FAQ](#faq---microsoft-defender-for-storage-pricing).
```

Eitan-Shteinberg committed:
Update articles/defender-for-cloud/defender-for-storage-introduction.md

Co-authored-by: Ben Mansheim <19514688+bmansheim@users.noreply.github.com>

Eitan-Shteinberg committed:
Update articles/defender-for-cloud/defender-for-storage-introduction.md

Co-authored-by: Ben Mansheim <19514688+bmansheim@users.noreply.github.com>

bmansheim commented on articles/defender-for-cloud/defender-for-storage-introduction.md:
```diff
@@ -23,6 +23,13 @@ Defender for Storage doesn't access the Storage account data and has no impact o
+>
+> Customers using the legacy per-transaction pricing plan need to migrate to the new per-storage account plan to access these new features and pricing. The legacy per-transaction pricing plan charges are based on the number of analyzed transactions in the storage account.
+>
+> For further details, please refer to the [Microsoft Defender for Storage FAQ](#faq---microsoft-defender-for-storage-pricing).
```
```suggestion
> For further details, please refer to the [Microsoft Defender for Storage FAQ](azure-defender-storage-configure.md#faq---microsoft-defender-for-storage-pricing).
```

Eitan-Shteinberg committed:
Update articles/defender-for-cloud/defender-for-storage-introduction.md

Co-authored-by: Ben Mansheim <19514688+bmansheim@users.noreply.github.com>

bmansheim commented on articles/defender-for-cloud/defender-for-storage-introduction.md:
```diff
@@ -23,6 +23,13 @@ Defender for Storage doesn't access the Storage account data and has no impact o
+>
+> Customers using the legacy per-transaction pricing plan need to migrate to the new per-storage account plan to access these new features and pricing. The legacy per-transaction pricing plan charges are based on the number of analyzed transactions in the storage account.
+>
+> For further details, please refer to the [Microsoft Defender for Storage FAQ](azure-defender-storage-configure.md#faq---microsoft-defender-for-storage-pricing).
```
```suggestion
> For further details, please refer to the [Microsoft Defender for Storage FAQ](../storage/common/azure-defender-storage-configure.md#faq---microsoft-defender-for-storage-pricing).
```

Eitan-Shteinberg committed:
Update articles/defender-for-cloud/defender-for-storage-introduction.md

Co-authored-by: Ben Mansheim <19514688+bmansheim@users.noreply.github.com>
